include/podio/Reader.h

```diff
@@ -2,8 +2,16 @@
 #define PODIO_READER_H
 
 #include "podio/Frame.h"
+#include "podio/FrameCategories.h"
 #include "podio/podioVersion.h"
 
+#include <memory>
+#include <stddef.h>
```

Comment:
Any reason why iwyu prefers `stddef.h` over `cstddef`?

Reply:
IIRC this is a long standing issue with iwyu that it adds C-headers when non-`std::` versions are used (e.g. `size_t` instead of `std::size_t`). There should be a compatibility list, so if `cstddef` is put manually iwyu shouldn't complain or propose to add `stddef.h`

Reply:
Now it's stddef